### PR TITLE
Create block to static rich text

### PR DESCRIPTION
**Pull request by augustolazaro**
This block won't be available on CMS, so its content will be static (in all languages as well).

### CODE REVIEW

File: react/Static.tsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { injectIntl } from 'react-intl'
+
+import { RichText } from './index'
+
+export default injectIntl(RichText)
```
Comment (klzns) [inline react/Static.tsx]: I would add a comment here to explain what is happening. It took me a while to understand how is this different from the usual rich-text.

Comment (augustolazaro) [reply react/Static.tsx]: I just wanna remove the schema, so it won't be available in to edit on CMS. It's a poor solution to  an uncommon situation

Comment (lbebber) [reply react/Static.tsx]: I agree with @klzns , add a comment on the code here explaining what this does and why this was done. I'm not sure I fully understand to be honest

**Comment by augustolazaro** [comment]
I found out a better solution, we don't need it anymore

**Pull request closed without merging**